NetSSL_OpenSSL/include/Poco/Net/HTTPSClientSession.h

```diff
@@ -144,6 +144,9 @@ class NetSSL_API HTTPSClientSession: public HTTPClientSession
 	// HTTPSession
 	void abort();
 
+	// HTTPSession
```

Comment:
What does this comment mean?

Reply:
They use the same comment for `void abort()` method, I assumed it points to the parent class that originally implements it, so I added it here too to keep convention.